textdb/textdb-sandbox/team3datafile.txt

```diff
@@ -0,0 +1,4 @@
+networkx 1.1.1
```

Comment:
If you don't want to be blocked by @sandeepreddy602 , please find a good place for your data file, such as java/edu/uci/ics/textdb/sandbox/team3/team3lucenequeryexample?